uvm/servlets/admin/config/network/Interface.js

```diff
@@ -585,6 +597,18 @@ Ext.define('Ung.config.network.Interface', {
                         emptyText: '[enter IPv4 address]'.t(),
                         allowBlank: false,
                         validator: function(value) {
+                            // Validation for not allowing network and broadcast address of selected netmask                           
+                            try {
+                                var staticPrefix = this.up("ungrid").getSelectionModel().getSelection()[0].get('staticPrefix'),
+                                    aliasNetMask = Util.getV4NetmaskMap()[staticPrefix];
+
+                                if(value == Util.getNetwork(value, aliasNetMask) || value == Util.getBroadcast(value, aliasNetMask)) {
+                                    return Ext.String.format('Entered address is network or broadcast address of selected Netmask {0}.'.t(), aliasNetMask);
```

Comment:
Let's reword as "Address cannot be a network or broadcast address with netmask {0}"

Reply:
Updated the error message verbiage